src/daft-plan/src/logical_ops/repartition.rs

```diff
@@ -31,7 +31,12 @@ impl Repartition {
     pub fn multiline_display(&self) -> Vec<String> {
         let mut res = vec![];
         res.push(format!("Repartition: Scheme = {:?}", self.scheme));
-        res.push(format!("Number of partitions = {}", self.num_partitions));
+        res.push(format!(
+            "Number of partitions = {}",
+            self.num_partitions
+                .map(|n| n.to_string())
+                .unwrap_or("None".to_string())
```

Comment:
Maybe "Unknown" instead?

```suggestion
                .unwrap_or("Unknown".to_string())
```